rustql-extractor/src/main.rs

```diff
@@ -29,163 +31,198 @@ extern crate syntax;
 
 pub mod visitor;
 
-use self::rustc_driver::driver::{CompileState, CompileController};
-
 use rustql_common::data;
 
 //use rustc_driver::driver::{CompileController, PhaseController, CompileState};
 use crate::rustc::hir;
 use crate::rustc::hir::intravisit::walk_crate;
+use rustc_driver::Compilation;
+use rustc_interface::interface;
+use rustc_tools_util::*;
```

Comment:
Is it necessary to have a wildcard import here?

Reply:
Fixed.